.github/ISSUE_TEMPLATE/--question.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: "❓ Question"
-about: Please use https://discuss.akka.io for questions
+about: Please use https://discuss.pekko.io for questions
```

Comment:
we don't have a discuss.pekko.io - could you use https://github.com/apache/incubator-pekko/discussions instead?

Reply:
ok, I'll fix that